### PR TITLE
README for pypi

### PR DESCRIPTION
**Pull request by PrimozGodec**
##### Issue

Readme for pypi is a bit out off date and the format is diferent than in other addons.

##### Description of changes
Updating readme for PyPI and changing the name of the file such is that it is consistent with other addons.

@ajdapretnar can you check this PR. Also, feel free to suggest any changes in the text.

##### Includes
- [ ] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: README.pypi [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Orange3 Image Analytics
+=======================
+
+Simplifies loading of images and through deep network-based embedding
```
Comment (ajdapretnar) [inline README.pypi]: embeddings?

File: README.pypi [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+enables their analysis. Embedding represents images with feature vector,
```
Comment (ajdapretnar) [inline README.pypi]: vectors?

File: README.pypi [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+allowing the use of Orange's standard arsenal of widgets for
+clustering, classification or any other kind of
+feature-based analysis. Image Analytics also provides visualization
+widgets for images and allow saving them.
```
Comment (ajdapretnar) [inline README.pypi]: allows (sure of this one)

**Comment by PrimozGodec** [comment]
@ajdapretnar thank you. :)

**Pull request merged**